lib/oxttl/src/turtle.rs

```diff
@@ -213,6 +214,75 @@ impl TurtleParser {
         }
     }
 
+    /// Creates a vector of iterators that may be used to parse a Turtle document slice in parallel.
+    /// To dynamically specify target_parallelism, use e.g. std::thread::available_parallelism
```

Comment:
nit:  use
```
[`std::thread::available_parallelism`]
```
 to get a link + add a dot at the end of the sentence

Reply:
Fixed!